### PR TITLE
Fixing simulate, pulling models from gplibrary.

### PR DESCRIPTION
**Pull request by 1ozturkbe**
Resolves the issue with posynomials in objective in simulate, but currently I'm breaking the simple_wing_SP by pulling from gplibrary. I know the models aren't identical, but I think we should use the one from gplibrary. @alisaab can you help me figure out the issue? My solutions do not converge when I run signomial_simple_wing/models.py.

### CODE REVIEW

**Comment by alisaab** [comment]
@1ozturkbe the solutions are not converging when I run gpkitmodels/SP/SimPleAC/SimPleAC_mission also.  has this been the case in the past ?

File: robust/signomial_simple_wing/models.py [hunk 1]
```diff
@@ -1,101 +1,9 @@
...
-                        W_f >= TSFC * T_flight * D]
-
-    # return Model(W_f/LoD, constraints)
-    return Model(D, constraints)
-    # return Model(W_f, constraints)
-    # return Model(W,constraints)
-    # return Model(W_f*T_flight,constraints)
-    # return Model(W_f + 1*T_flight*units('N/min'),constraints)
-
+from gplibrary.SP.SimPleAC.SimPleAC_mission import SimPleAC, Mission
```
Comment (alisaab) [inline robust/signomial_simple_wing/models.py]: Maybe this should be from gplibrary.SP.SimPleAC.SimPleAC_mission import SimPleAC, Mission by gpkitmodels.SP.SimPleAC.SimPleAC_mission import SimPleAC, Mission

File: robust/signomial_simple_wing/models.py [hunk 2]
```diff
@@ -1,101 +1,9 @@
 from gpkit import Variable, Model, SignomialsEnabled, VarKey, units, Vectorize
 import numpy as np
-
 from robust.simulations import simulate, read_simulation_data
 import os
 
-
-def simple_wing_sp():
```
Comment (alisaab) [inline robust/signomial_simple_wing/models.py]: Probably we should keep both this simple_wing_sp and add the new SimPleAC.

**Comment by 1ozturkbe** [comment]
@alisaab this is why it would have been preferable to figure out what the issue is before merging, but I guess we can do that now regardless, but on master.

**Pull request merged**